### PR TITLE
media-sound/loudgain: Add new ebuild

### PR DESCRIPTION
**Pull request by haasn**
Useful tool for mass-tagging ReplayGain tags to a diverse number of file
formats. The version I chose to package is a fork of ghedo's upstream
project which was last touched in 2014 and therefore seems abandoned.

Signed-off-by: Niklas Haas <git@haasn.xyz>

### CODE REVIEW

File: media-sound/loudgain/loudgain-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+fi
+
+inherit cmake-utils
+
+DESCRIPTION="ReplayGain 2.0 loudness normalizer based on the EBU R128/ITU BS.1770 standard"
+HOMEPAGE="https://github.com/Moonbase59/loudgain"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE=""
```
Comment (ConiKost) [inline media-sound/loudgain/loudgain-9999.ebuild]: If empty, drop it.

File: media-sound/loudgain/loudgain-0.6.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+fi
+
+inherit cmake-utils
+
+DESCRIPTION="ReplayGain 2.0 loudness normalizer based on the EBU R128/ITU BS.1770 standard"
+HOMEPAGE="https://github.com/Moonbase59/loudgain"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE=""
```
Comment (ConiKost) [inline media-sound/loudgain/loudgain-0.6.8.ebuild]: If empty, drop it.

File: media-sound/loudgain/loudgain-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/Moonbase59/${PN}.git"
+	inherit git-r3
```
Comment (ConiKost) [inline media-sound/loudgain/loudgain-9999.ebuild]: Shouldn't be ```inherit``` before EGIT_*?

File: media-sound/loudgain/loudgain-0.6.8.ebuild [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/Moonbase59/${PN}.git"
+	inherit git-r3
```
Comment (ConiKost) [inline media-sound/loudgain/loudgain-0.6.8.ebuild]: Shouldn't be ```inherit``` before EGIT_*?

Comment (haasn) [reply media-sound/loudgain/loudgain-0.6.8.ebuild]: I don't think the order matters. But I can change it if you prefer.

Comment (haasn) [reply media-sound/loudgain/loudgain-0.6.8.ebuild]: Seems about random which line comes first, in current ebuilds.

**Comment by ajakk** [comment]
Unfortunately new leaf packages aren't being added by proxy-maint anymore, so I'm closing this. Fortunately, it looks like this has been added to the [GURU overlay](https://wiki.gentoo.org/wiki/Project:GURU) in the meantime!

**Pull request closed without merging**